draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -627,10 +627,34 @@ in cache before performing any followup queries.
 With these optimizations in place, and conforming DNS servers,
 using SVCB does not add network latency to connection setup.
 
+Moreover, if an A or AAAA response arrives before an SVCB response, the
+client SHOULD wait up to CD milliseconds before connecting as if the
+SVCB query returned NODATA, but MUST NOT transmit any information that
+could be altered by an SVCB response until the SVCB response arrives.
+For example, a TLS ClientHello may be altered by the "esnikeys" value
+of an SVCB response {{svcparamkeys-esnikeys}}. CD (Connection Delay)
+is a configurable parameter. The recommended value is 50 milliseconds,
+as per the guidance in {{!HappyEyeballsV2=RFC8305}}.
+
+An SVCB record is consistent with an active or in-progress connection C
+if the A or AAAA address for its SvcDomainName matches the destination
```

Comment:
This definition is good but it leaves out ports and IP hints.

Reply:
I can add ports, but I don't think IP hints are part of the definition. (Or, rather, they don't change things.) There's two cases for hints:

1. The hints match C's address: Continue using C's address.
2. The hints do not match C's address: The definition of the hints says that C's address should be used since it corresponds to the A/AAAA of SvcDomainName.

So I left them out and just included the port.